### ISSUE TITLE
TypeScript errors in @types/applepayjs/index.d.ts

### ISSUE THREAD

**Issue by pellebjerkestrand** [Adyen/adyen-web#318]
**Describe the bug**
Trying to compile a Typescript project referencing this module results in the following error.

```bash
node_modules/@types/applepayjs/index.d.ts:9:39 - error TS2689: Cannot extend an interface 'EventTarget'. Did you mean 'implements'?

9 declare class ApplePaySession extends EventTarget {
                                        ~~~~~~~~~~~

node_modules/@types/applepayjs/index.d.ts:780:25 - error TS2304: Cannot find name 'Node'.

780         composedPath(): Node[];
                            ~~~~
```

**To Reproduce**
1. `import` this module anywhere in a codebase that compiles with TypeScript.
2. Compile
3. See error

**Expected behavior**
I expect the types to be OK, so that the project referencing this module can compile.

**Screenshots**
![image](https://user-images.githubusercontent.com/2156573/93787376-b3006a00-fc30-11ea-9f81-66decaaf6598.png)

**Desktop (please complete the following information):**
 - OS: macOS Catalina
 - Runtime: NodeJS v12.18.0
 - Version: 3.13.1

**Additional context**
TypeScript 4.0.2

**Comment by pellebjerkestrand** [comment]
TLDR: add `dom` to `tsconfig.json`'s `compilerOptions.lib`:

```json
{
  "compilerOptions": {
    "lib": ["dom"]
  }
}
```

I guess this might be unique to our situation as we have a single large project that contains both our server (TS) and our client (JS). Our `tsconfig.json` therefore does not include `dom` in its `compilerOptions.lib` array. `dom` includes the `Node` interface, and also allows extension of `EventTarget`.

Adding `dom` to `compilerOptions.lib` solves this problem. It adds some new ones, but I guess that's our problem.
